README.md

```diff
@@ -1 +1,103 @@
+- Copy this file to any machine that would like to use the library
+- Install the library using `install.packages` command:
+  - `install.packages(path_to_file, repos = NULL, type="source")`
+- Note: If installing the library from its .tar.gz file is desired, please provide `samples@osisoft.com` with details on the use case and feedback so we can discuss alternate hosting platforms for this file (such as [CRAN](https://cran.r-project.org/)).
+
+
+#### Run the Sample
+- Open [Sample.R](Sample.R) and set the data view configuration to point to the desired data view and time
+  - By default, this sample is pointing to the bulk upload data view in the time range that contains data
+  - If this sample is being used to retrieve other data, change this section's configuration accordingly
+- Step through [Sample.R](Sample.R) and confirm the data frame is successfully created
+
+## Running the automated test
+
+### Test Using RStudio
+
+The unit tests for this sample were built using [testthat](https://cran.r-project.org/web/packages/testthat/index.html).
+- The authentication test confirms that a token of non-trivial length is returned
+- The data view interpolated data test confirms that the correct data is received for the specified time period. This is done by obtaining the data frame from the library, then using the [all.equal](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/all.equal) function to compare it against a data frame generated from a locally-stored csv representing the "correct" answer. This style of test ensures that data retrieval and pagination are working.
+
+To run the automated tests:
+- Open the library in RStudio
+- Prepare the `sampleoutput.csv` for comparison (this repo's .gitignore prevents any sampleoutput.csv file from being checked in)
+  - If using Wind Turbine Data
+    - Rename the provided [sampleoutput_placeholder.csv](sampleoutput_placeholder.csv) file to `sampleoutput.csv`
+    - Confirm that the [Bulk Upload Sample](https://github.com/osisoft/sample-ocs-bulk_upload-dotnet) has been run and the necessary data and data view are in your OCS Namespace
+  - If using a different data set
+    - Obtain a csv of the data view output for this period using a tool of your choice
+      - For example, use this sample to generate a data frame, visually inspect it for accuracy, then save it to csv using [write.csv](https://www.rdocumentation.org/packages/AlphaPart/versions/0.8.1/topics/write.csv)
+    - Save this csv as `sampleoutput.csv` in the sample's root folder
+- Replace the values prefixed with `test_only_` in the `config.yml` file as necessary.
+  - Note: The placeholder file preloads these data view values for use with the [Bulk Upload Sample](https://github.com/osisoft/sample-ocs-bulk_upload-dotnet) wind turbine data
+- Execute the tests in the R terminal
+  - library(devtools)
+  - use_testthat
+  - test()
+
+---
+
+Tested against R 4.0.3
```

Comment:
We are currently stating this twice, I think we can leave this one and delete the comments from the ## Requirements section. ("Developed against" and "Tested against" doesn't really need to be distinguished, IMO)

Reply:
I agree it seems redundant and I removed it, but it looks like a majority of our samples have it this way. I just spot checked 3 of them and all 3 have it "developed against" at the top and "tested against" in the footer

Reply:
Yeah, I suppose you're right. I'd like to standardize on just saying it once though.